modin/core/io/text/text_file_dispatcher.py

```diff
@@ -1023,24 +1040,46 @@ def _read(cls, filepath_or_buffer, **kwargs):
             )
 
         is_quoting = kwargs["quoting"] != QUOTE_NONE
+        usecols = kwargs["usecols"]
         use_inferred_column_names = cls._uses_inferred_column_names(
-            names, skiprows, kwargs.get("skipfooter", 0), kwargs.get("usecols", None)
+            names, skiprows, kwargs["skipfooter"], usecols
```

Comment:
```suggestion
            names, skiprows, kwargs.get("skipfooter", 0), usecols
```
why it was changed?

Reply:
Same as in https://github.com/modin-project/modin/pull/5482#discussion_r1093774064